apps/payments/next/app/[locale]/[offeringId]/checkout/[interval]/[cartId]/processing/page.tsx

```diff
@@ -0,0 +1,20 @@
+import { getApp, LoadingSpinner } from '@fxa/payments/ui/server';
+import { DEFAULT_LOCALE } from '@fxa/shared/l10n';
+import { headers } from 'next/headers';
+
+export default async function ProcessingPage() {
+  const locale = headers().get('accept-language') || DEFAULT_LOCALE;
+  const l10n = getApp().getL10n(locale);
+
+  return (
+    <section className="flex flex-col" data-testid="payment-processing">
+      <div className="flex flex-col text-center text-sm">
```

Comment:
Could we eliminate the nested div by making the section's classname `flex flex-col text-center text-sm` instead?

Reply:
Ah yes much better. Thank you.